{{ cookiecutter.package_name|slugify }}/{% if cookiecutter.continuous_integration == "GitLab" %}.gitlab-ci.yml{% endif %}

```diff
@@ -127,8 +127,9 @@ Publish:
     - .docker
   stage: deploy
   variables:
+    DOCKER_PUSH: 1
     DOCKER_BASE_IMAGE: $CI_REGISTRY_IMAGE/ci:$CI_IMAGE_SHA
-    DOCKER_IMAGE: $CI_REGISTRY_IMAGE
+    DOCKER_IMAGE: $CI_REGISTRY_IMAGE/app
```

Comment:
I don't think this is necessary: the image may be the name of the repository itself (which I think is preferably because then the image name is that of the actual solution – otherwise with several projects you end up with a bunch of images all called `app`).

Reply:
That's what I thought as well, but it looks like gitlab needs to create a repository inside the registry of the project if we don't provide one. If we don't provide any, it will name it `Root image` and our repositories will look like that in our gitlab's project registry: 
![image](https://user-images.githubusercontent.com/18347608/154943779-cf5bbdb4-1a39-41a4-aa49-d057ccbc6cd3.png)

Would you rather keep the default `Root image` repository, or name it `app` or something else which makes it easy to understand it's the application repository of the project?

Reply:
In practice `Root image` only shows in the gitlab UI. The repository is actually at the root of the registry. So I can remove it if you still think that makes more sense.

Reply:
Oki, I removed it anyways. Should be good to merge :)